populate_course_run_reporting_type_and_announcement.py

```diff
@@ -0,0 +1,80 @@
+import argparse
```

Comment:
the root directory has a hyphen, I think for that reason I wasn't able to do a relative or absolute import, is there any way to do that other than changing the structure of the whole repo?  
I don't know all the different jenkins jobs running these scripts and changing the root directory would break all of them